### PR TITLE
fix(docs): correct bibtex name order

### PR DESCRIPTION
**Pull request by Jonas-Eppard**
firstname and lastname of citation were reversed

Signed-off-by: Jonas Eppard <jonas.eppard@de.bosch.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,11 +38,11 @@ specific version) you can use the following BibTex entry:
 ```bibtex
 @software {carbynestack,
   title = {Carbyne Stack},
-  author = {Becker Sebastian and Duplys Paul and Graf Johannes and
-   Graffi Kalman and Grassi Alessandro and Greven David and Grewe Julian and
-   Jain Shalabh and Klenk Timo and Matyunin Nikolay and Modica Hanna and
-   Raskin Vadim and Scherer Petra and Suschke Volker and Trieflinger Sven and
-   Vlasakiev Veselin and Weinfurtner Jared},
+  author = {Sebastian Becker and Paul Duplys and Johannes Graf and
```
Comment (sbckr) [inline README.md:41]: We usually order contributors (authors) by last name using lexicographical order. I'd prefer to stick with this practice and keep the list ordered.
If you want to make sure first- and lastname are picked up correctly, the authors list should be updated like the following:
```
author = {Becker, Sebastian AND Duplys, Paun AND Graf, Johannes, [...]
}
``` 
as described [here](https://bibtex.eu/de/fields/author/)

**Review by sbckr — CHANGES_REQUESTED** [review]
Please keep authors in lexicographical order
(see comment)

File: README.md (hunk 1 above)
Comment (Jonas-Eppard) [reply README.md]: adressed in 8df7bb9

**Review by sbckr — APPROVED** [review]
LGTM

**Pull request merged**